app/reconciler/createChangeSetFromCompareStructures.ts

```diff
@@ -0,0 +1,92 @@
+import { isEqual } from 'lodash';
+import logger from '~/lib/logger.server';
+
+import { DnsRecordType } from '@prisma/client';
+
+import type { Change } from '@aws-sdk/client-route-53';
+import type { ReconcilerCompareStructure } from './ReconcilerTypes';
+
+interface InputType {
+  dbStructure: ReconcilerCompareStructure;
+  route53Structure: ReconcilerCompareStructure;
+}
+
+const createChangeSetFromCompareStructures = ({ dbStructure, route53Structure }: InputType) => {
+  const toChange: Change[] = [];
+
+  /**
+   * If something is present in route53 but is completely missing in db,
+   * then we have to delete it
+   */
+  Object.keys(route53Structure).forEach((fqdn) => {
+    Object.keys(route53Structure[fqdn]).forEach((type) => {
+      const route53Value = route53Structure[fqdn][type as DnsRecordType]!;
+      const dbValue = dbStructure[fqdn]?.[type as DnsRecordType];
+
+      if (dbValue) {
+        // Both of them have this
+        return;
+      }
+
+      toChange.push({
+        Action: 'DELETE',
```

Comment:
To confirm, you don't need `MultiValueAnswer` on a `DELETE`?  I think not, but wanting to check.

Reply:
No, during testing it worked perfectly without it

Reply:
Actually, you were right, it DID need it. The documentation does not mention it, upsert works without it, the returned error is complete nonsense (misses attributes that I do actually send), but sending MultiValue for the delete solved it. 

Nice catch, saved me a lot of debug time!